integration-tests/tasks/rhtap-install.yaml

```diff
@@ -32,7 +32,7 @@ spec:
         secretName: rhtap-cli-install
   steps:
     - name: install
-      image: quay.io/rhtap/rhtap-e2e:latest
+      image: quay.io/redhat-user-workloads/rhtap-shared-team-tenant/rhtap-e2e/rhtap-e2e@sha256:ffd08a9c0bfae255fbfd3e08cb434c381931342046f91352ff5277c53c1443dc
```

Comment:
I'm sorry I provided you wrong URL. I think this one should be correct:
```
quay.io/redhat-user-workloads/rhtap-shared-team-tenant/rhtap-e2e/rhtap-e2e:bc690d27a9031aa7bbbbc55ef54d6ea319d6738e
```

Reply:
Once it's updated, feel free to force push, as this is the type of change PR check won't test (without hassle of creating temp commit changing branch of the task in pipeline definition here: https://github.com/redhat-appstudio/rhtap-cli/blob/main/integration-tests/pipelines/rhtap-cli-e2e.yaml#L141)